src/dsp_tools/commands/validate_data/validate_data.py

```diff
@@ -130,38 +117,25 @@ def _save_graphs(filepath: Path, rdf_graphs: RDFGraphs) -> Path:
     cprint(f"\n   Saving graphs to {new_directory}   ", color="light_blue", attrs=["bold", "reverse"])
     generic_filepath = new_directory / filepath.stem
     rdf_graphs.ontos.serialize(f"{generic_filepath}_ONTO.ttl")
-    rdf_graphs.cardinality_shapes.serialize(f"{generic_filepath}_SHACL_CARD.ttl")
-    rdf_graphs.content_shapes.serialize(f"{generic_filepath}_SHACL_CONTENT.ttl")
-    rdf_graphs.data.serialize(f"{generic_filepath}_DATA.ttl")
-    onto_data = rdf_graphs.data + rdf_graphs.ontos
+    shacl_onto = rdf_graphs.shapes + rdf_graphs.ontos
+    shacl_onto.serialize(f"{generic_filepath}_SHACL_ONTO.ttl")
+    rdf_graphs.shapes.serialize(f"{generic_filepath}_SHACL.ttl")
+    rdf_graphs.data_onto.serialize(f"{generic_filepath}_DATA.ttl")
+    onto_data = rdf_graphs.data_onto + rdf_graphs.ontos
```

Comment:
*issue* This is redunant because `data_onto` already contains the `ontos`.

```suggestion
    onto_data = rdf_graphs.data_onto 
```

Probably you can inline `onto_data` without loosing clarity of the rest of the code.